frontend/src/views/shop/detail/DetailSitespeed.vue

```diff
@@ -3,10 +3,19 @@
         Sitespeed monitoring is not enabled for this shop. Please enable it in the shop settings to view performance metrics.
     </Alert>
     <div class="mb-1">
-        <a v-if="shop && shop.sitespeed.length > 0" class="btn btn-primary-outline" :href="'/sitespeed/' + shop.id + '/index.html'" target="_blank">
+        <a v-if="shop && shop.sitespeed && shop.sitespeed.length > 0" class="btn btn-primary-outline" :href="'/sitespeed/' + shop.id + '/index.html'" target="_blank">
             <i class="fa fa-chart-line"></i> View Sitespeed Report
         </a>
     </div>
+    <div v-if="shop && shop.sitespeed && shop.sitespeed.length > 0" class="panel panel-table chart-panel">
```

Comment:
Please remove the `panel-table` class. This class fully removes padding inside the panel and the chart have no room